src/core/connectionHelpers/baseConnectionHelper.js

```diff
@@ -44,8 +44,7 @@ export default class BaseConnectionHelper {
     }
     this.isStarted = true;
     this.startConnectionTokenPolling(
-      true, 
-      this.getConnectionTokenExpiry()
+      true
```

Comment:
Why did we take the connection token expiry out here?

Reply:
Hi John

This seems to be null on the first call resulting in an expiry = 0 which causes the timeout on the poll to execute immediately. By removing this the default for the polling takes the constant